Gems/Atom/Feature/Common/Assets/ShaderResourceGroups/PostProcessing/ViewSrg.azsli

```diff
@@ -26,8 +26,9 @@ partial ShaderResourceGroup ViewSrg
 
         // circle of confusion to screen ratio;
         float m_cocToScreenRatio;
+        [[pad_to(16)]]
```

Comment:
Is this HLSL syntax or AZSL only? I don't think I've seen this before

Reply:
Azsl only. It was recently added to azslc to automagically add the required padding instead of doing it explicitly and make a mistake.